lib/crash_course_agent.ex

```diff
@@ -0,0 +1,35 @@
+defmodule CrashCourseAgent do
+  use GenServer
+  @moduledoc """
+  Gen Server with Agent API
+  """
+  # Client
+  def start_link(state \\ %{}) do
+    GenServer.start_link(__MODULE__, state, name: __MODULE__)
+  end
+
+  def get(), do: GenServer.call(__MODULE__, {:get})
```

Comment:
Tuple consisting of one element is kinda redundant :)
```diff
- def get(), do: GenServer.call(__MODULE__, {:get})
+ def get(), do: GenServer.call(__MODULE__, :get)
```
The fix in the respective `handle_call` is required to match this as well.